Cognite.Simulator.Utils/ModelLibraryBase.cs

```diff
@@ -393,16 +393,24 @@ await _cdfSimulatorResources.UpdateSimulatorModelRevisionParsingStatus(
 
         private void UpdateModelParsingInfo(T modelState, SimulatorModelRevision modelRevision)
         {
-            if (modelState.ParsingInfo == null || modelState.ParsingInfo.LastUpdatedTime < modelRevision.LastUpdatedTime)
+            var status = modelRevision.Status;
+            var needsUpdate = modelState.UpdatedTime < modelRevision.LastUpdatedTime;
+            if (needsUpdate && status == SimulatorModelRevisionStatus.unknown)
+            {
+                _logger.LogDebug("Resetting download attempts counter for {ModelExtid} due to unknown status",
+                    modelState.ModelExternalId,
+                    modelState.ParsingInfo,
```

Comment:
is the logger able to properly print the `ParsingInfo `object?

Reply:
this is not used, and you are right